# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/IDBCursor.h
# Commit: fc5e1bee1332
# Full Hash: fc5e1bee13321946704a915ef4c7c80c0941dcb3
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-10-02 03:38:52
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Send two records with every ObjectStoreCursorResponse. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43252
# ==============================================================================

diff -r 63cf0966cb41 -r fc5e1bee1332 dom/indexedDB/IDBCursor.h
--- a/dom/indexedDB/IDBCursor.h	Tue Oct 01 09:15:47 2019 +0000
+++ b/dom/indexedDB/IDBCursor.h	Tue Oct 01 09:15:49 2019 +0000
@@ -120,6 +120,8 @@
 
   IDBCursorDirection GetDirection() const;
 
+  bool IsContinueCalled() const { return mContinueCalled; }
+
   void GetKey(JSContext* aCx, JS::MutableHandle<JS::Value> aResult,
               ErrorResult& aRv);
 
@@ -159,6 +161,8 @@
     mBackgroundActor = nullptr;
   }
 
+  void InvalidateCachedResponses();
+
   NS_DECL_CYCLE_COLLECTING_ISUPPORTS
   NS_DECL_CYCLE_COLLECTION_SCRIPT_HOLDER_CLASS(IDBCursor)
 